PyDSS/dssTransformer.py

```diff
@@ -79,3 +85,8 @@ def GetValue(self, VarName, convert=False):
         else:
             return None
         return VarValue
+
+    def SetActiveObject(self):
+        self._dssInstance.Circuit.SetActiveElement(self._FullName)
+        if self._dssInstance.CktElement.Name() != self._dssInstance.Element.Name():
+            raise InvalidParameter('Object is not a circuit element')
```

Comment:
This method is already implemented in the parent class, so you should be able to delete it.  The logic looks the same.

Reply:
removed